quick-start/src/test/resources/integration-test-data/input/input-es-trips.json

```diff
@@ -0,0 +1,56 @@
+{
+  "envelope": {
+    "attachments": {
+      "id": "123",
+      "name": "bob smith",
+      "salary": "1000"
+    },
+    "headers": {},
+    "instance": {
+      "e2eentity": {
+        "id": "123",
+        "name": "bob smith",
+        "salary": 1000
+      },
+      "info": {
+        "title": "e2eentity",
+        "version": "0.0.1"
+      }
+    },
+    "triples": [
+      {
+        "triple": {
+          "subject": {
```

Comment:
these are not valid RDF triples -- does it matter?

Reply:
It doesn't matter to the current tests. I have copied all the input test resources from datahub core test/resources. Probably might be useful when expanding the tests.